scripts/deploy-support.sh

```diff
@@ -22,7 +22,7 @@ deploy() {
   service=$1
   pushd services/$service
   install_deps
-  serverless deploy  --stage $stage
+  serverless deploy  --stage $stage --enforce-hash-update
```

Comment:
I think we don't actually need this, based on the other stuff we wound up digging into with the API endpoints.